src/patternfly/utilities/Flex/examples/flex-align-content-example.hbs

```diff
@@ -1,4 +1,4 @@
-{{#> title titleType="h2" pf-c-title--Modifier="pf-m-lg pf-m-margin"}}
+{{#> title titleType="h2" title--modifier="pf-m-lg pf-m-margin"}}
```

Comment:
we can remove the pf-m-margin here it's no longer used in the title component. This is for every title in the flex exmaples.

Reply:
fixed